lib/data/multipleBackendGateway.js

```diff
@@ -30,6 +31,19 @@ const multipleBackendGateway = {
                 hashedStream.pipe(writeStream);
             }
         }
+
+        if (keyContext.tagging) {
+            const validationTagRes = parseTagFromQuery(keyContext.tagging);
+            if (validationTagRes instanceof Error) {
+                const log = createLogger(reqUids);
+                log.debug('tag validation failed', {
```

Comment:
yeah, that's kind of what I thought, but I copied that bit from the metadataStoreObject function